pkg/shell/shell.html

```diff
@@ -2120,7 +2164,7 @@ <h4 class="modal-title">Run Image</h4>
                 </td>
               </tr>
               <tr id="containers-run-image-cpu">
-                <td translatable="yes">CPU priority</td>
+                <td><label class="control-label" translatable="yes">CPU priority</label></td>
```

Comment:
Out of scope of the issue being fixed here. Feel free to do this as a follow up.